packages/lib/sdk/package.json

```diff
@@ -78,6 +78,12 @@
 			"import": "./src/plugins/index.js",
 			"require": null,
 			"types": "./src/plugins/index.js"
+		},
+		"./plugins/discoverPluginPackageRootPathSync": {
```

Comment:
Using a dedicated export, as needs to be required from a `commonjs` context, and `esm` only modules are included in the main `./plugins` export